fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/plan/temp/MemoExpressionTest.java

```diff
@@ -220,7 +218,11 @@ public boolean equals(Object o) {
 
         @Override
         public int hashCode() {
-            return Objects.hash(identity, childRefs.stream().map(ExpressionRef::get).collect(Collectors.toList()));
+            return Objects.hash(identity,
+                    quantifiers.stream()
+                            .map(Quantifier.ForEach::getRangesOver)
+                            .map(ExpressionRef::get)
```

Comment:
Remove the `::get` this should just work with the rangesOver as the hashCode is delegated to the ref it ranges over.